alf/algorithms/sac_algorithm.py

```diff
@@ -846,6 +857,30 @@ def _calc_critic_loss(self, info: SacInfo):
         When the reward is multi-dim, the entropy reward will be added to *all*
         dims.
         """
```

Comment:
The comment above has been separated from the entropy reward code.

Reply:
Now make the comment embedded to the right place.